main.py

```diff
@@ -59,14 +60,13 @@ def main():
         if next_valid_location.empty:
             # Add HOME location to the end of the dataframe
             travel_df = travel_df.append(travel_df.iloc[0])
```

Comment:
Consider using `home_location` variable instead of comment + travel_df.iloc[0]
Would it be more readable?

Reply:
Fixed